_layouts/base.html

```diff
@@ -113,13 +115,31 @@ <h1><a href="{{ site.baseurl }}/">{{ site.title }}</a></h1>
             window.location.assign("/index_" + lang);
         }
       }
+
+      function loadAnchors(){
+        anchors.options.placement = 'left';
+        anchors.add('#page > h2, #page > h3, #page > h4, #page > h5, #page > h6');
+      }
+
+      function loadSearch(lang, site) {
+        docsearch({
+          apiKey:'a57ef92bf2adfae863a201ee43d6b5a1',
+          indexName: 'brew_all',
+          inputSelector: '#search-bar',
+          algoliaOptions: { 'facetFilters': [
+            'lang: ' + lang,
+            'site: ' + site,
+          ] },
+          debug: true
+        });
+      }
     </script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.2.0/anchor.min.js" 
             integrity="sha256-m1eTvwEHwmsw4+XKF7BshClVJEPwTVycveNl0CS0Mkk=" 
-            crossorigin="anonymous"></script>
-    <script type="text/javascript">
-      anchors.options.placement = 'left';
-      anchors.add('#page > h2, #page > h3, #page > h4, #page > h5, #page > h6');
-    </script>
+            crossorigin="anonymous"
+            onload="loadAnchors()"></script>
```

Comment:
I'm confused about how this change is related (if it is)? If not: could it go in another PR?